projects/test-app/src/app/state/state.module.ts

```diff
@@ -1,9 +1,9 @@
 import { CommonModule } from '@angular/common';
 import { ModuleWithProviders, NgModule, Optional, SkipSelf } from '@angular/core';
-import { NgrxAutoEntityModule } from '@briebug/ngrx-auto-entity';
+import { NGRX_AUTO_ENTITY_APP_STORE, NgrxAutoEntityModule } from '@briebug/ngrx-auto-entity';
```

Comment:
Looks like NGRX_AUTO_ENTITY_APP_STORE and Store got moved to app.module, so we don't need to import here